### DISCUSSION TITLE
Sonoff TH10 as Heater and Temperature and Humidity Sensor

### DISCUSSION THREAD

**Discussion by mpelas199** [RavenSystem/esp-homekit-devices#1666] [MEPLHAA Scripts]
Hello, I have a Sonoff TH10 paired with a Si7021 sensor and I wanted to use it as a Heater and as a Temperature and Humidity Sensor at the same time. This is the code I came up with, but it doesn't seem to be working. 

```
{
"c": { "l": 13 },
"a": [{
  "t":25,
  "s":5,
  "g":14,
  "0": {"r":[{"g":12}]},
  "1": {"a":0},
  "3": {"r":[{"g":12,"v":1}]},
}, {
  "t": 24,
  "g": 14,
  "n": 4,
  "j": 30,
  "z": -0.8,
  "5":{"m":[[0,0.02]]}
}]
}
```
Im not sure if it is even possible, but if it is can anyone help?

**Comment by xtremedays** [comment]
I use mine as a 'thermostat'

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"t":21,"n":3,"s":5,"g":14,"w":3,"0":{"r":[{"g":12}]},"1":{"a":0},"3":{"r":[{"g":12,"v":1}]},"4":{"r":[{"g":12,"v":1}]}}]}
